tools/gstswitchcontroller.c

```diff
@@ -401,12 +269,9 @@ gst_switch_controller_on_new_connection (GDBusServer * server,
     GDBusConnection * connection, gpointer user_data)
 {
   GstSwitchController *controller = GST_SWITCH_CONTROLLER (user_data);
-  GstSwitchAddNewClientParams *params = NULL;
   guint register_id = 0;
   GError *error = NULL;
 
```

Comment:
I'd probably be good to have some sort of logging when a new client is connected.